packages/generator/src/options/options.ts

```diff
@@ -220,14 +209,14 @@ export const generatorOptionsCli = {
   },
   readme: {
     describe:
-      'When set to true, the generator will write a README.md file into the root folder of every package. This option does not make that much sense without also set useSwagger to "true".',
+      "When set to true, the generator will write a README.md file into the root folder of every package. The information in the readme are mostly derived from accompanying Swagger or OpenAPI files. Therefore it is recommended to use the 'readme' option in combination with 'useSwagger'.",
     type: 'boolean',
     default: false,
     hidden: true
   },
   writeReadme: {
     describe:
-      'When set to true, the generator will write a README.md file into the root folder of every package. This option does not make that much sense without also set useSwagger to "true".',
+      "When set to true, the generator will write a README.md file into the root folder of every package. The information in the readme are mostly derived from accompanying Swagger or OpenAPI files. Therefore it is recommended to use the 'writeReadme' option in combination with 'useSwagger'.",
```

Comment:
[q] If this is merged in main - should this option be removed?

Reply:
If so also look for other deprecated options in the list.

Reply:
Or is this done in this ticket: https://github.com/SAP/cloud-sdk-backlog/issues/978?

Reply:
> Or is this done in this ticket: [SAP/cloud-sdk-backlog#978](https://github.com/SAP/cloud-sdk-backlog/issues/978)?

Yes, exactly